scenes/startup.txt

```diff
@@ -123,4 +126,4 @@
 		All tests passed.
 *else
 	*comment allow user to select individual tests, view logs etc.
-	*goto_scene _test_entry main
\ No newline at end of file
+	*goto_scene _test_entry main
```

Comment:
Something odd is going on with the line endings I think?

Reply:
This seems to be the whole Windows (CRLF) vs Unix (LF) thing. I work on a Mac (which is of course really Unix).
I haven't worked out a way of dealing with this yet.

Reply:
I have configured git so that any LF endings will be converted to CRLF on commit but there still seem to be a few problems.

Reply:
No worries, I'll push a fix before we merge it :)